scripts/analysis.py

```diff
@@ -0,0 +1,378 @@
+    num_dict: dictionary
+        dictionary of number of reactors progression
+        with country_government as key
+    """
+
+    power_dict = collections.OrderedDict({})
+    num_dict = collections.OrderedDict({})
+
+    for gov in governments:
+        capacity = []
+        num_reactors = []
+        cap = 0
+        count = 0
+        for num in timestep:
+            for enter in entry:
+                if enter[3] == num and enter[2] == gov[1]:
+                    cap += enter[0]
+                    count += 1
+            for dec in exit:
+                if dec[3] == num and dec[2] == gov[1]:
+                    cap -= dec[0]
+                    count -= 1
+            capacity.append(cap)
+            num_reactors.append(count)
+
+        power_dict[gov[0]] = np.asarray(capacity)
+        num_dict[gov[0]] = np.asarray(num_reactors)
+
+    return power_dict, num_dict
+
+
+def stacked_bar_chart(dictionary, timestep, xlabel, ylabel, title, outputname):
+    """ Creates stacked bar chart of timstep vs dictionary
+
+    Parameters
+    ----------
+    dictionary: dictionary
+        holds time series data
+    timestep: array
+        array of timestep (x axis)
+    xlabel: string
+        xlabel of plot
+    ylabel: string
+        ylabel of plot
+    title: string
+        title of plot
+
+    Returns
+    -------
+    Stacked bar chart
+    """
+
+    # set different colors for each bar
+    color_index = 0
+    top_index = True
+    prev = ''
+    plot_array = []
+    # for every country, create bar chart with different color
+    for key in dictionary:
+        label = key.replace('_government', '')
+        # very first country does not have a 'bottom' argument
+        if top_index is True:
+            plot = plt.bar(1950+(timestep/12), dictionary[key], .5,
+                           color=cm.viridis(1.*color_index/len(dictionary)),
+                           edgecolor='none', label=label)
+            prev = dictionary[key]
+            top_index = False
+        # from the second country has 'bottom' argument
+        else:
+            plot = plt.bar(1950 + (timestep/12), dictionary[key], .5,
+                           color=cm.viridis(1.*color_index/len(dictionary)),
+                           edgecolor='none', bottom=prev, label=label)
+            prev += dictionary[key]
+
+        plot_array.append(plot)
+        color_index += 1
+
+    # plot
+    plt.ylabel(ylabel)
+    plt.title(title)
+    plt.xlabel(xlabel)
+    plt.legend(loc=(1.0, 0))
+    plt.grid(True)
+    plt.savefig(outputname, format='png', bbox_inches='tight')
+
+
+def plot_power(filename, cursor):
+    """ Gets capacity vs time for every country
+        in stacked bar chart.
+
+    Parameters
+    ----------
+    filename: file
+        cyclus output file used
+    cursor: cursor
+        cursor for sqlite3
+
+    Returns
+    -------
+    stacked bar chart of net capapcity vs time
```

Comment:
this function doesn't actually return anything. If it did, you would format this like:

```python
"""
...

Returns
-------
matplotlib.figure
    stacked bar chart of net capapcity vs time
"""
```